fearless/Modules/Staking/StakingUnbondSetup/StakingUnbondSetupViewController.swift

```diff
@@ -0,0 +1,207 @@
+import UIKit
+import SoraFoundation
+import CommonWallet
+
+final class StakingUnbondSetupViewController: UIViewController, ViewHolder {
+    typealias RootViewType = StakingUnbondSetupLayout
+
+    let presenter: StakingUnbondSetupPresenterProtocol
+
+    var selectedLocale: Locale {
+        localizationManager?.selectedLocale ?? .autoupdatingCurrent
+    }
+
+    init(
+        presenter: StakingUnbondSetupPresenterProtocol,
+        localizationManager: LocalizationManagerProtocol
+    ) {
+        self.presenter = presenter
+
+        super.init(nibName: nil, bundle: nil)
+
+        self.localizationManager = localizationManager
```

Comment:
Why does this particular order matter?

Reply:
The problem here is that ```presenter``` is local variable so it must be initialised before calling super. ```LocalizationManager``` is requirement of the protocol with default value and it must be initialised after super